samples/bluetooth/mesh/sensor_server/README.rst

```diff
@@ -82,7 +83,10 @@ The models are used for the following purposes:
 
 The model handling is implemented in :file:`src/model_handler.c`.
 It uses the TEMP_NRF5 or BME680 temperature sensor depending on the platform.
-The :ref:dk_buttons_and_leds_readme library is used to detect button presses.
+The sample has a descriptor related to the :c:var:`bt_mesh_sensor_present_dev_op_temp` sensor, which specifies tolerance values for the ``TEMP_NRF5`` temperature sensor calculated based on the ``nRF52832 Product Specification <--- Link somehow``.
```

Comment:
You could ask Mia Koen if she knows a way to ling the specification.

Reply:
Awaiting an answer!